resources/user-agents/browsers/firefox-focus/firefox-focus-android-1-0-on.json

```diff
@@ -37,6 +37,19 @@
             "Android_5_1", "Android_5_0",
             "Android"
           ]
+        },
+        {
+          "match": "Mozilla/5.0 (*Linux*Android?9) applewebkit* (*khtml*like*gecko*) Version/* Focus/#MAJORVER#.#MINORVER#* Chrome/* Mobile Safari/*",
+          "platforms": [
+            "Android_9_0"
```

Comment:
Dont we have a placeholder for this Android version?

Reply:
This is the only Android 9 placeholder I found in `platforms.json`: https://github.com/browscap/browscap/blob/master/resources/platforms.json#L3985